### DISCUSSION TITLE
Retrieving the input of another field

### DISCUSSION THREAD

**Discussion by Ninerian** [rjsf-team/react-jsonschema-form#4173] [Q&A]
Hello,

currently I try to implement a form with two fields, where the value defined in the first field changes the data in the second field. The data is retrieved via an api call in the custom widgets, which is used for the second field. Unfortunately have I not found a solution to pass the data to between fields.

I tried the following solutions:

1. Dynamic dependency
    The scheme has a dependency defined where for each possible value in field 1, the title of the second value is set. This works and the component get re-rendered. I can set the value via a property. But the component for field 2 gets recreated when I choose a value and also the chosen value is not selected

2. Setting a const in the properties 
    I changed the scheme of point 1 to add a const property which contains the value of field 1. This is now available in the component and the selecting works now. Unfortunately do I now retrieve an error that the value should be equal to const. 

Do you have any ideas how to solve this issue?

**Comment by heath-freenome** [comment]
Do you have a codesandbox.io example?
